service.rb

```diff
@@ -33,7 +34,14 @@ def initialize(port)
         end
 
         response.redirect('/show/data', status: 303)
-      
+
+      elsif request.get? && request.path == "/daily_data.json"
+        response.content_type = "text/plain"
```

Comment:
Is `text/plain` the right media type for a [JSON document](https://tools.ietf.org/html/rfc8259)?